ott/core/segment.py

```diff
@@ -12,11 +12,13 @@
 # limitations under the License.
 """Prepare point clouds for parallel computations."""
 
-from typing import Optional, Tuple
+from typing import Callable, Optional, Tuple
 
 import jax
 from jax import numpy as jnp
 
+from ott.geometry import costs
```

Comment:
using the suggestion in `bar_problems` above to pass directly the padding vector, this import is no longer needed.